platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## 3.5.1
+
+### Other changes
```

Comment:
Nit: normally we don’t break out patch releases by topic, since they’re so small. This will be pretty easy to address right before release when we copy-edit release notes anyhow.